CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.10.0 (Released 2025-02-28)
+
+BREAKING CHANGES
+
+- transfers: switch to newer `/accounts/{accountID}`-prefixed endpoints
+- transfers: update transfer methods to accept an accountID parameter
+
```

Comment:
usually, we update the changelog after the PR is merged -- cause this release can contain other updates as well -- e.g. cancellations

Reply:
done